Graviton_vs_x86_monitoring_differences.md

```diff
@@ -0,0 +1,42 @@
+# x86 monitoring differences
```

Comment:
can you please elaborate it? x86 vs aarch64 profiler tool differences or something like that

Reply:
really more different architectures have different tools because they have different features

Reply:
Done.